experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -535,8 +539,12 @@ def save(self, commit=True):
         experiment.status_next = self.status_next
         experiment.publish_status = self.publish_status
         experiment.save()
+        self.post_save_actions(experiment)
         return experiment
 
+    def post_save_actions(self, experiment):
+        pass
```

Comment:
Oh yeah, the method override pattern totally works here, but also you can just do this with overriding save

```py
    def save(self, *args, **kwargs):
        experiment = super().save(*args, **kwargs)
        <some post save actions here>
        return experiment
```